lumen/ai/agents.py

```diff
@@ -447,7 +447,7 @@ class SQLAgent(LumenBaseAgent):
         }
     )
 
-    provides = param.List(default=["table", "sql", "pipeline", "data"], readonly=True)
```

Comment:
I think the requires part is correct, but agents should list what they provide fully. Right now we don't have agents that would require a table but not a pipeline but in theory we could, so the provides list should list everything.